docker/development/compose.yml

```diff
@@ -12,8 +12,8 @@ services:
       - POSTGRES_DB=documenso
     ports:
       - 54320:5432
-#    networks:
- #      - my-custom-created-network
+    # networks:
```

Comment:
Why are there changes for networks?

Reply:
Oversight on my part, will correct indentation before pushing to uat.